electron/config.mjs

```diff
@@ -0,0 +1,79 @@
+import { app } from 'electron';
+import { getPort } from 'get-port-please';
+import { join, dirname, parse } from 'path';
+import log from 'electron-log/main.js';
+import util from 'util';
+import { renameSync } from 'fs';
+
+// App config
+const dev = !app.isPackaged;
+const appName = 'Acorn';
+const appDir = app.getAppPath();
+const logsDir = app.getPath('logs');
+const resourcesDir = dirname(appDir);
+const dataDir = join(app.getPath('userData'), appName);
+const threadsDir = process.env.THREADS_DIR || join(dataDir, 'threads');
+const workspaceDir = process.env.WORKSPACE_DIR || join(dataDir, 'workspace');
+const port = process.env.PORT || !dev
+    ? await getPort({ portRange: [30000, 40000] })
+    : 3000;
```

Comment:
nit: can we use parentheses to make the order of operations a bit less ambiguous here?

```suggestion
const port = process.env.PORT || (!dev
    ? await getPort({ portRange: [30000, 40000] })
    : 3000);
```

I think this should work the same, if JavaScript and whatever combination of linters we have allows it.